### ISSUE TITLE
Immediate crash after welcome screen display

### ISSUE THREAD

**Issue by dinana** [imperador/chromefy#33]
Hi,

Using R69 versions of arnold and recovery (eve & caroline) I created an image using method 1, flashed it on a usb drive and booted from it. On my old Samsung XE700T1C it boots and works beautifully, I can even install it to the disk and multiboot with Android and windows.

However on my Dell XPS 15 9560 as soon as the "G" Logo is finished and the welcome screen is displayed (the one with the enable debugging functionality link) the screen turns black then the laptop halts.

Unfortunately I have no more info and don't really know how I can get more logs, tried looking into the USB drive's state partition for any logs but couldn't find any, let me know if there's a way to debug/log what is happenning.

Thanks

**Comment by alesimula** [comment]
Have you tried with any other Chromium builds other than ATB (Ex: FydeOS, ot the custom builds i posted on the XDA thread like CloudRadish) to see if they all have the same issue?

**Comment by hello2clans** [comment]
Hi, I also have the same problem. If I used Arnold v69 with caroline and eve, it works fine. 
With v71 of nocturne, I tried all images ArnoldTheCat, Emyde and Cloudradish all from v 1 to 3 along with ArnoldTheBat v72. All works fine as base, but after chromefy, it crashes on welcome screen and laptop just turns off. Except, CloudRadish v1 worked fine... it doesn't crash, but its not stable on my laptop.

**Comment by alesimula** [comment]
By method i mean the TPM workatround (TPM1.2 replacement or TPM2 emulator); The chromefying methods (1, 2A, 2b, 3) don't actually matter and generate the same result in the end.

By CloudRadish v1 do you mean the first image named CloudRadish (CloudRadishNut.bin)?
If so, why was it unstable?

**Comment by dinana** [comment]
In my case I used method 1 with caroline image.

**Comment by alesimula** [comment]
Nope, CloudRadishNut is actually pretty vanilla with very few drivers

**Comment by hello2clans** [comment]
Since the kernel from v69 of Arnold works really well for me, is it possible to compile vtpmproxy.ko(not exact name) kernel module for v69 of Arnolds kernel ?

That way, we will be able to eliminate the suspicion of crash occuring because of upstreamed kernel (v72 as compared to v69)

**Comment by alesimula** [comment]
Nope, you could just try chromefying v72 with TPM1.2 replacement method; if it still crashes, the TPM2 emulator's most likely not the issue

**Comment by dinana** [comment]
> Nope, you could just try chromefying v72 with TPM1.2 replacement method; if it still crashes, the TPM2 emulator's most likely not the issue

That I did, ATB v72, eve and caroline v70, created the img via method 1, flashed to usb and when I boot from it it crashes and halts just after the Google logo disappears and shows the first welcome screen.

Also, ATB v72 works fine, no wifi though, but when I tried disabling wifi from BIOS and booting the chromefied USB I had the same issue, tried disabling TPM from BIOS with no luck too.

Any idea why this happens? or how to debug it?

**Comment by dinana** [comment]
Also tried nocturne v71 restore image with no luck.

**Comment by hello2clans** [comment]
I tested bunch of v71 recoveries (including Caroline) and all of them crashes on boot. Which means, something is changed by Google in v71 which causes the crash. Anyway to debug this issue ?

**Comment by alesimula** [comment]
Try checking logs in /var/log/

**Comment by hello2clans** [comment]
Well, I just tested v71 Caroline with fydeos v5.31 and it works ! So, I don't know what if something is worth with Arnold v72 or tpm... 

Dinana, try Caroline v71 with fydeos it works for me.

**Comment by ars0nboi** [comment]
I also have Dell XPS 15 9560. As Dinana mentioned, mine does the exact same. Did hello2clans suggestion work for you Dinana? hello2clans, do you have a Dell XPS 15 9560 with nvidia graphics?

**Comment by hello2clans** [comment]
Hi ars0nboi. Only Caroline v71 with fydeos works for v71. Swtpm does not work with my laptop for some reason (crashes at welcome screen). I am still trying to figure why swtpm does not work. I have a Acer Swift 3 without Nvidia.

**Comment by quendo1997** [comment]
> I also have Dell XPS 15 9560. As Dinana mentioned, mine does the exact same. Did hello2clans suggestion work for you Dinana? hello2clans, do you have a Dell XPS 15 9560 with nvidia graphics?

same with me. I have a dell xps 9560 and i can't get i past booting to the setup screen. seconds later the screen turns black and shuts down.

**Comment by quendo1997** [comment]
Dell XPS 15 9560 works with the latest version of fydeos. I am happy now. --> RESOLVED!

**Comment by ars0nboi** [comment]
Got mine working fydeOS 6.0, nocture 72, and Caroline v72. Everything works except the mic. --Dell XPS 15 9560. Any suggestions would be useful.

**Comment by vannhi** [comment]
Hi,
I have the same issue with an XPS 13 9343 and an HP Pavilion 14-bk001. Thanks to someone here, I found that ATB 72 + lulu 73 + swtpm works with both. However there is no crostini with lulu, so I'm interested in finding another codename with crostini which can work with these laptops.

Anyway to retrieve the logs? I'm testing using live USB.

**Comment by vannhi** [comment]
It seems that the problem comes from TPM. My two laptops have Intel PTT as TPM 2.0.
I have finally got my HP Pavilion working with fydeOS 6.1, nocturne 73 and swtpm but other actions are required: The TPM should be activated and not hidden in BIOS, and clearing TPM in BIOS may be needed. Otherwise, it crashes.

**Comment by buhlaze** [comment]
ugh, now i can't boot with the new fydeos 7.0 chromified. I'm out of options. I don't know what they changed.

**Comment by dragon788** [comment]
@vannhi you should try the kefka image instead of lulu, it should work similarly but has crostini.

**Comment by vannhi** [comment]
Thank you @dragon788 , I will test it. I can't do it now because I just returned my laptop for repair.

**Comment by vannhi** [comment]
@dragon788 I just received my laptop back from repair, so I tested kefka r74 as you advised:
- ATB Special  Kefka: crash for 2 laptops
- Fydeos 7.1 + kefka: works for HP (like nocturne), but crash for the Dell :(

**Comment by flaep** [comment]
Same here.
Thinkpad Helix (20CG)
I tried  the folling combinatons using the first methog on ubuntu:

Camd64OS_R74-11895.B-Special + chromeos_11895.118.0_eve_recovery_stable-channel_mp + swtpm

Camd64OS_R74-11895.B-Vanilla + chromeos_11895.118.0_eve_recovery_stable-channel_mp + chromeos_11895.118.0_caroline_recovery_stable-channel_mp 

I also tried set the security chip from intel PTT to dicrete TPM and reset the TPM in windows and BIOS/EFI
Tried to boot to a chromegfied installer after every change.

When i open the stick in Linux, /var/logs is empty. Where are the log files on chromeos?

The later combination works on a HP 840 G1.
